packages/fx-core/src/core/middleware/utils/v3MigrationUtils.ts

```diff
@@ -168,3 +170,45 @@ export async function readAndConvertUserdata(
 
   return returnAnswer;
 }
+
+export async function updateAndSaveManifestForSpfx(
+  context: MigrationContext,
+  manifest: string
+): Promise<void> {
+  const remoteTemplatePath = path.join(AppPackageFolderName, MANIFEST_TEMPLATE_CONSOLIDATE);
+  const localTemplatePath = path.join(AppPackageFolderName, "manifest.template.local.json");
+
+  const contentRegex = /\"\{\{\^config\.isLocalDebug\}\}.*\{\{\/config\.isLocalDebug\}\}\"/g;
+  const remoteRegex = /\{\{\^config\.isLocalDebug\}\}.*\{\{\/config\.isLocalDebug\}\}\{/g;
+  const localRegex = /\}\{\{\#config\.isLocalDebug\}\}.*\{\{\/config\.isLocalDebug\}\}/g;
```

Comment:
An easier way might be getting the manifest with mustache render (For local manifest, render with `{config.isLocalDebug: true}` and for remote manifest, render with `{config.isLocalDebug: false}`). Both ways are OK for me.